components/StarRating.js

```diff
@@ -0,0 +1,27 @@
+import Ratings from 'react-ratings-declarative';
+
+
+const StarRating = ({rating, size, spacing}) => {
+  return (
+    <Ratings
+      rating={rating}
+      typeOfWidget='star'
+      widgetRatedColors='rgb(255,165,0)'
+      widgetEmptyColors='#cccccc'
+      widgetDimensions={size}
+      widgetSpacings={spacing}
+    >
+      <Ratings.Widget style={{border:'1px solid red'}} />
+      <Ratings.Widget />
+      <Ratings.Widget />
+      <Ratings.Widget />
+      <Ratings.Widget />
+      </Ratings>
+  )
+}
+StarRating.defaultProps = {
+    rating:0,
+    size:'1.2rem',
+    spacing:'.2rem',
+}
+export default StarRating
```

Comment:
New star rating component, takes in three props, 
- rating: number (value of the rating) 
- size: string (how large the stars will be)
- spacing: string (how far apart each star will be)